portfolio/src/main/webapp/nickname.jsp

```diff
@@ -0,0 +1,29 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Angeline Chen</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons">
+    <link rel="stylesheet" href="https://unpkg.com/bootstrap-material-design@4.1.1/dist/css/bootstrap-material-design.min.css" integrity="sha384-wXznGJNEXNG1NFsbm0ugrLFMQPWswR3lds2VeinahP8N0zJw9VWSopbjv2x7WCvX" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-white">
+        <a class="nav-link" href="/data">Back</a>
+    </nav>
+    <main>
+        <form action="/nickname" method="POST">
```

Comment:
This seems to be mostly a duplicate of the form in data.jsp.  What's the purpose of each of these?  Without a screenshot or a way to see your site, it's hard for me to understand what's going on here.  Maybe some HTML comments would help?

Reply:
The form in data.jsp creates a comment.  The form in nickname.jsp changes the nickname.  I commented the code in the servlet that renders the templates.  Should I also comment the code in the templates?

Reply:
That would be really helpful, yes.  It looks like you can use `<%-- --%>` to add JSP comments that will be stripped when the page is rendered.